hist/histv7/inc/ROOT/RHistData.hxx

```diff
@@ -417,13 +522,41 @@ public:
 
    // FIXME: Add a way to query the inner data
 
-   /// Merge with other RHistDataMomentUncert data, assuming same bin configuration.
+   /// Add an other RHistDataMomentUncert data, assuming same bin configuration.
    void Add(const RHistDataMomentUncert& other) {
+      assert(fMomentXW.size() == other.fMomentXW.size());
+      assert(fMomentX2W.size() == other.fMomentX2W.size());
       for (size_t d = 0; d < DIMENSIONS; ++d) {
          fMomentXW[d] += other.fMomentXW[d];
          fMomentX2W[d] += other.fMomentX2W[d];
       }
    }
+
+   /// Multiply by a scalar.
+   template <typename T>
+   void MultiplyByScalar(const T scalar) {
+      for (size_t d = 0; d < DIMENSIONS; ++d) {
```

Comment:
This is a perfect for loop: the compiler will optimize this to death for all reasonable `DIMENSIONS`.